janus.c

```diff
@@ -5197,6 +5197,15 @@ gint main(int argc, char *argv[])
 					JANUS_LOG(LOG_INFO, "Setting event handlers statistics period to %d seconds\n", period);
 				}
 			}
+
+			item = janus_config_get(config, config_events, janus_config_type_item, "combine_media_stats");
+			if(item && item->value) {
+				gboolean combine = janus_is_true(item->value);
+				janus_ice_event_set_combine_media_stats(combine);
+				if(combine)
+					JANUS_LOG(LOG_INFO, "Event handler configured to send media stats combined in a single event\n");
+			}
+
```

Comment:
Can you remove the extra empty lines around this block?

Reply:
T-3 :)